mne/time_frequency/tfr.py

```diff
@@ -505,7 +505,8 @@ def _preproc_tfr(data, times, freqs, tmin, tmax, fmin, fmax, mode,
     from ..viz.utils import _setup_vmin_vmax
 
     copy = baseline is not None
-    data = rescale(data, times, baseline, mode, copy=copy)
+    if mode is not None:
+        data = rescale(data, times, baseline, mode, copy=copy)
```

Comment:
To me, this seems like the appropriate place to do the checking. This can basically be rephrased to say ``if rescale: do rescale``. But the example is weird. What is the point of setting ``baseline=(-0.5, 0), mode=None``?